docs/install/docker-plugin.rst

```diff
@@ -0,0 +1,74 @@
+
+========================================
+Installing the Flocker Plugin for Docker
+========================================
+
+Before installing the Flocker plugin for Docker, you will need to have installed Flocker on some nodes, using the instructions in the previous topics.
+
+On the same machine where you ran ``flocker-ca`` while installing Flocker, you will need to generate a new API user certificate and key for a user named ``plugin``, as described in the :ref:`generate-api` instructions.
+
+Upload these files to :file:`/etc/flocker/plugin.key` and :file:`/etc/flocker/plugin.crt` on the nodes where you want to run the Flocker plugin for Docker.
+
+You will also need to have Docker (at least version 1.8 or later) installed. The following command will install the latest version available:
+
+.. prompt:: bash $
+
+   wget -qO- https://get.docker.com/ | sudo sh
```

Comment:
Better to link to Docker's installation docs than to reproduce any specific instructions here that may be subject to change, or vary between platforms.